static/app/actionCreators/onboardingTasks.tsx

```diff
@@ -11,7 +11,7 @@ import {OnboardingTask, OnboardingTaskStatus, Organization} from 'sentry/types';
  */
 export function updateOnboardingTask(
   api: Client | null,
-  organization: Organization,
+  organization: Organization | null,
```

Comment:
@roggenkemper why do we need to handle `null` here? When don't we have an org?

Reply:
in guideStore.tsx , all the attributes that are coming off of organization in setActiveOrganization (https://github.com/getsentry/sentry/blob/4b7b97730f4b692564b7ebeb7433e09387caa165/static/app/stores/guideStore.tsx#L123-L127)  have a check to see if Organization is null and if it is, set that attribute (orgSlug, orgId) as null too. since that's where this organization is coming from,  i added the case to handle it being null.

Reply:
@roggenkemper let's just not even call `updateOnboardingTask` if there is no org then, that's preferable